### ISSUE TITLE
Create project mission

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#50]
## User story
1. As a Team
2. I want a good project mission
3. So that we have a clear purpose for this project

## Acceptance criteria
* [ ] Upload project mission to planning docs
* [ ] Update Github README

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
